src/poliastro/maneuver.py

```diff
@@ -141,7 +145,9 @@ def hohmann(cls, orbit_i, r_f):
 
         t_trans = np.pi * np.sqrt(a_trans ** 3 / k)
 
-        return cls((0 * u.s, dv_a), (t_trans, dv_b))
+        return cls(
+            (tof.decompose(), dv_a.decompose()), (t_trans.decompose(), dv_b.decompose())
```

Comment:
Shouldn't this be `t_trans + tof`? (Did not think about it in depth, please confirm)